portfolio/src/main/webapp/script.js

```diff
@@ -54,15 +54,24 @@ function fetchComments() {
       const div = document.createElement("div");
       div.setAttribute("class", "comment");
       
+      const userInfo = document.createElement("div");
+      userInfo.setAttribute("class", "user-info");
+
+      const picture = document.createElement("img");
+      picture.setAttribute("class", "comment-picture");
+      picture.src = comment.pictureURL;
+
       const user = document.createElement("div");
       user.setAttribute("class", "user");
-      user.textContent = comment.user + " said:";
+      user.textContent = comment.user;
```

Comment:
I think `createTextNode` would be a better match, textContent is not really meant for this.